source/_integrations/zwave_js.markdown

```diff
@@ -950,3 +954,15 @@ Set the log level for `zwave_js_server` to `debug`. This can either be done in y
 ##### Disable Z-Wave JS logging manually, or via an automation
 
 Set the log level for `zwave_js_server` to a level higher than `debug`. This can either be done in your `configuration.yaml` in the `logger` section, or using the `logger.set_level` action. The Z-Wave JS logs will no longer be included in the Home Assistant logs, and if the log level of Z-Wave JS was changed by the integration, it will automatically change back to its original level.
+
+## Unsupported functionality
+
+This sections lists functionality that is available in Z-Wave but that is not currently supported in Home Assistant.
+
+### Setting the controller into learn mode to receive network information
+
+In Home Assistant, it is currently not possible to set the Z-Wave controller into learn mode to receive network information from another controller.
+
+### Include and exclude the device in an existing network using [classic inclusion](#classic-inclusion-versus-smartstart)
```

Comment:
This is still confusing. Including/excluding in an existing network is the regular thing that works fine.

Reply:
To clarify, this is to satisfy the following documentation-related certification requirement:
> For Z-Wave controller Role Types, the documentation MUST describe:
1. How to include and exclude the device in an existing network using classic inclusion.
2. How to include and exclude other devices.
3. How to put the controller into learn mode to receive network information from another controller.

HA/Z-Wave JS can **only** do 2. We do not support **being** included in another existing network (1.) and because of that we also do not support receiving network information from another controller (3.)

Reply:
Right. We need to clarify what "device" means here. In 1 we're talking about the controller as a device.

Reply:
Thank you for clarifying.